internal/grpc/services/ocmshareprovider/ocmshareprovider.go

```diff
@@ -265,7 +265,7 @@ func (s *service) CreateOCMShare(ctx context.Context, req *ocm.CreateOCMShareReq
 
 	ocmshare := &ocm.Share{
 		Token:         tkn,
-		Name:          filepath.Base(info.Path),
+		Name:          req.ResourceId.OpaqueId[len("fileid-/home"):],
```

Comment:
Why this? The name of the share was decided to be the filename of the shared resource...

Reply:
This is a stop gap, @yasharpm will improve on it, see https://github.com/pondersource/nc-sciencemesh/issues/296#issuecomment-1538222765

Reply:
It's done now. I just don't have access to push to the branch.

Reply:
I reverted it as per https://github.com/pondersource/nc-sciencemesh/issues/296#issuecomment-1540682653